rosserial_embeddedlinux/CMakeLists.txt

```diff
@@ -1,7 +1,13 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(rosserial_embeddedlinux)
 
-find_package(catkin REQUIRED)
+find_package(catkin REQUIRED COMPONENTS
+  geometry_msgs
+  nav_msgs
+  rosserial_client
+  sensor_msgs
+  std_msgs
```

Comment:
These aren't necessary at build time. The file to fix here is the `package.xml`.